requirements.txt

```diff
@@ -1,4 +1,4 @@
-alembic==1.6.2
+alembic==1.4.1
```

Comment:
Why downgrade?

Reply:
I think it was some changes I made to debug alembic migration but I will change it back because I do not think the experiment was successful.